src/network.ts

```diff
@@ -0,0 +1,65 @@
+// Copyright 2016-2019, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as aws from "@pulumi/aws";
+
+import { EnforcementLevel, ResourceValidationPolicy, validateTypedResource } from "@pulumi/policy";
+
+import { registerPolicy } from "./awsGuard";
+import { defaultEnforcementLevel } from "./enforcementLevel";
+
+// Mixin additional properties onto AwsGuardArgs.
+declare module "./awsGuard" {
+    interface AwsGuardArgs {
+        albHttpToHttpsRedirection?: EnforcementLevel;
+    }
+}
+
+// Register policy factories.
+registerPolicy("albHttpToHttpsRedirection", albHttpToHttpsRedirection);
+
+/** @internal */
+export function albHttpToHttpsRedirection(enforcementLevel: EnforcementLevel): ResourceValidationPolicy {
+    return {
+        name: "alb-http-to-https-redirection-check",
```

Comment:
Do we want the `-check` suffix? If we keep it in the name, should we also include it in the JS/TS idiomatic camel cased name? Note that we already have a handful of policies that includes the "Check" suffix in the JS/TS name. We should decide the naming pattern we want to use and follow it consistently.

https://github.com/pulumi/pulumi-policy-aws/blob/7796f71a27838a33cc96dfdd2f9b8824a4419a0f/src/compute.ts#L29

https://github.com/pulumi/pulumi-policy-aws/blob/7796f71a27838a33cc96dfdd2f9b8824a4419a0f/src/database.ts#L27-L29

https://github.com/pulumi/pulumi-policy-aws/blob/7796f71a27838a33cc96dfdd2f9b8824a4419a0f/src/database.ts#L32

https://github.com/pulumi/pulumi-policy-aws/blob/7796f71a27838a33cc96dfdd2f9b8824a4419a0f/src/storage.ts#L41